iron-image.html

```diff
@@ -396,8 +389,14 @@
       },
 
       _resolveSrc: function(testSrc) {
-        var baseURI = /** @type {string} */(this.ownerDocument.baseURI);
-        return (new URL(Polymer.ResolveUrl.resolveUrl(testSrc, baseURI), baseURI)).href;
+        var resolved = Polymer.ResolveUrl.resolveUrl(testSrc, this.$.baseURIAnchor.href);
```

Comment:
Are you sure this works even with a `<base>` tag? e.g. polygit.

I _believe_ that is why we did [this](https://github.com/PolymerElements/platinum-sw/pull/112/files#diff-eb52a757ba28ccb094696ec2bfe241e4R110) in platinum-sw.